modules/core/src/v2/coins/eth.ts

```diff
@@ -832,6 +869,48 @@ export class Eth extends BaseCoin {
     }).call(this).asCallback(callback);
   }
 
+  /**
+   * Return types, values, and total amount to send in a batch transaction, using the method signature
+   * `distributeBatch(address[], uint256[])`
+   * @param recipients {EthRecipient[]} transaction recipients
+   * @returns {batchExecutionInfo} the types, values, and total amount to execute the batch transaction
+   */
+  static getBatchExecutionInfo(recipients: EthRecipient[]): batchExecutionInfo {
+    const addresses = [];
+    const amounts = [];
+    let sum = new BigNumber(0);
+    _.forEach(recipients, (recipient) => {
+      addresses.push(recipient.address);
+      amounts.push(recipient.amount);
+      sum = sum.plus(new BigNumber(recipient.amount));
+    });
+
+    return {
+      types: ['address[]', 'uint256[]'],
+      values: [addresses, amounts],
+      totalAmount: sum.toFixed(0)
+    };
+  };
+
+  /**
+   * Create new recipients array to point to Batcher contract with correct amount and also construct data needed for
+   * Batcher execution
+   * @param recipients {EthRecipient[]} prebuilt transaction recipients
```

Comment:
Remove types from doc comments (these are now read from the typescript types).

Reply:
Fixed.